src/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import '../App.css'
+
+class Footer extends Component {
+  render() {
+    return (
+      <div id="Footer">
```

Comment:
https://caniuse.com/#search=footer
you can safely replace this with the html 5 <footer> semantic tag.
This will help readability.